### PR TITLE
Fish Shell

### PR DESCRIPTION
**Pull request by bswinnerton**
**Shell**
- [x] Installer
- [x] `.zsh` (or now, `.sh`) autosourcer

**Prompt**
- [ ] Newline after the output of any command

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,17 +24,16 @@ There are a few special files and conventions to point out:
 
 - `bin/`: Anything in bin/ will be added to your `$PATH` and be made available
   to execute anywhere in your shell.
-- `topic/*.zsh`: Any files ending in .zsh get loaded into your environment.
-- `topic/path.zsh`: Any file named path.zsh is loaded first and is expected to
+- `topic/*.auto.sh`: Any files ending in .auto.sh get loaded into your
```
Comment (bswinnerton) [inline README.md]: Let's add another line item for `*.auto.fish`.

File: fish/config.fish [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# Autoload any files that end in auto.sh
+for file in ~/.dotfiles/**/*.auto.sh
+  sh $file
+end
+
+# Execute Archey
+~/.dotfiles/bin/archey
+
+# Resource FZF to ensure keybindings are set on top of ZSH
+sh ~/.dotfiles/fzf/fzf.auto.sh
```
Comment (bswinnerton) [inline fish/config.fish]: Can this be removed now?

File: vim/vim.symlink/plugin/plugins.vim [hunk 3]
```diff
@@ -30,6 +30,7 @@ Plug 'pangloss/vim-javascript'
 Plug 'jparise/vim-graphql'
 Plug 'zerowidth/vim-copy-as-rtf'
 Plug 'fatih/vim-go'
+Plug 'dag/vim-fish'
```
Comment (bswinnerton) [inline vim/vim.symlink/plugin/plugins.vim:33]: This plugin is terribly slow. Let's remove it.

File: vim/vim.symlink/plugin/vim-tmux-navigator.vim [hunk 4]
```diff
@@ -0,0 +1,2 @@
+" Workaround for slow startup time with Fish: https://git.io/vS5cq
+set shell=/bin/bash\ -i
```
Comment (bswinnerton) [inline vim/vim.symlink/plugin/vim-tmux-navigator.vim]: Is this necessary in conjunction with [this line](https://github.com/bswinnerton/dotfiles/pull/11/files#diff-edbe3b236fe65b305ca6cbc2f5fa585dR3)?

File: README.md (hunk 1 above)
Comment (bswinnerton) [reply README.md]: Added in 78ade07.

File: vim/vim.symlink/plugin/vim-tmux-navigator.vim (hunk 4 above)
Comment (bswinnerton) [reply vim/vim.symlink/plugin/vim-tmux-navigator.vim]: It isn't. Removed in ff751cc6508d49309809020ab45e2815b6759a50.

File: vim/vim.symlink/plugin/plugins.vim (hunk 3 above)
Comment (bswinnerton) [reply vim/vim.symlink/plugin/plugins.vim:33]: This doesn't appear to be the case any more.

**Comment by bswinnerton** [comment]
Shoutout to @toddmazierski for first getting me interested in Fish. 

👋  Hi Todd! Let's meet up sometime!

**Comment by toddmazierski** [comment]
Hey, @bswinnerton! Delighted to hear you've made the switch! I'll have to mine your config for goodies. 😄 

Yes, let's meet up soon! 🍻

**Pull request merged**